src/java.base/share/classes/jdk/internal/util/HexDigits.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Hello Volkan, this should be `2024, 2025, `

Reply:
Fixed in f94af95d8d0.